package.json

```diff
@@ -12,8 +12,7 @@
     "@ministryofjustice/frontend": "^1.0.1",
     "agentkeepalive": "^4.2.0",
     "applicationinsights": "^2.2.1",
-    "applicationinsights-native-metrics": "^0.0.7",
```

Comment:
removed this as I don't think it's worked on much and it was dependant on a vulnerable version of ansiregex I believe. I remember it being sat on 0.0.6 for a while with a vulnerability as well. Don't think we actually make use of any of the metrics it provides.